src/did/util.ts

```diff
@@ -0,0 +1,40 @@
+import { sha256 } from "@cosmjs/crypto";
+import { DIDDocument, DataWithSeq } from "../proto/panacea/did/v2/did";
+import Long from "long";
+import { Secp256k1 } from "../crypto/secp256k1";
+
+const bs58 = require('bs58');
+
+export class DidUtil {
+  static getDid(pubKeyCompressed: Uint8Array): string {
+    return `did:panacea:${bs58.encode(sha256(pubKeyCompressed))}`;
+  }
+
+  static getPublicKeyBase58(pubKeyCompressed: Uint8Array): string {
+    return bs58.encode(pubKeyCompressed);
+  }
+
+  static signDidDocument(privKey: Uint8Array, didDocument: DIDDocument, sequence: Long = Long.fromInt(0)): Uint8Array {
+    const dataWithSeq: DataWithSeq = {
+      data: DIDDocument.encode(didDocument).finish(),
+      sequence: sequence,
+    };
+    return Secp256k1.sign(sha256(DataWithSeq.encode(dataWithSeq).finish()), privKey);
+  }
+
+  static signDid(privKey: Uint8Array, did: string, sequence: Long = Long.fromInt(0)): Uint8Array {
+    const didDocument: DIDDocument = {
```

Comment:
`JSONStringOrStrings` type is undefined and Arrays type is `[]`.(which is actually the same.) 🤔

Reply:
If I understand your comment correctly, the `contexts` cannot be `[]` because it's not an array. Instead, the `contexts` is an object type `Strings` which contains a `string[]`. 

The `JSONStringOrStrings` is a type defined in Go in the `panacea-core`, for the conversion between protobuf and json.

Reply:
IMHO, I just found that it would be better to sign on the full DID document instead of just DID. It's because we'll anyway query a DID Document and sequence from Panacea in order to get the sequence :).
But then, we have to change the logic in the `panacea-core`... So, let's not touch it now.

Reply:
> If I understand your comment correctly, the `contexts` cannot be `[]` because it's not an array. Instead, the `contexts` is an object type `Strings` which contains a `string[]`.
> 
> The `JSONStringOrStrings` is a type defined in Go in the `panacea-core`, for the conversion between protobuf and json.

Oh, I was mistaken.

Reply:
> By the way, I just found that it would be better to sign on the full DID document instead of just the DID. It's because we'll anyway query a DID from Panacea in order to get the sequence :).
> I missed that point when I design it initially :).
> But then, we have to change the logic in the `panacea-core`... So, let's not touch it now.

Yes, Let's revice this later.